### PR TITLE
Remove unused

### PR DESCRIPTION
**Pull request by robxyy**

### CODE REVIEW

File: lib/android/build.gradle [hunk 1]
```diff
@@ -38,7 +38,5 @@ dependencies {
   compile 'com.android.support:support-annotations:25.2.0'
   compile 'com.android.support:design:25.2.0'
   compile 'com.fasterxml.jackson.core:jackson-databind:2.8.3'
-  compile 'com.facebook.fresco:fresco:0.11.0'
```
Comment (gpeal) [inline lib/android/build.gradle]: Can you put this in a separate PR?
It'll also conflict with this: https://github.com/airbnb/native-navigation/pull/107

Comment (robxyy) [inline lib/android/build.gradle]: Done. can we remove fresco dependencies? because react native has been dependent on it: [build.gradle#L282](https://github.com/facebook/react-native/blob/master/ReactAndroid/build.gradle#L282), if it's good enough I would be happy to submit a PR.

**Comment by gpeal** [comment]
@robxyy I don't feel comfortable removing fresco as a dependency because we actually use Fresco in ReactBottomNavigation and ReactToolbar so it should be an explicit dependency.
